src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -80,9 +89,16 @@
 
     public Configuration(){}
 
+    @Deprecated
+    public Configuration(CharSource staticSource, CharSink staticSink){
```

Comment:
Then I don't understand your PR. A configuration is linked to a static space where it is stored. By moving to `CharSource` / `CharSink` this can be more than just a file. Why do you want it completely dynamic?